docs_app/content/tutorial/3-creating-an-observable.md

```diff
@@ -0,0 +1,140 @@
+# 3. Creating An Observable
+
+## TLDR:
+
+- You should probably use a provided [creation function](API).
+- Create an observable with the `new Observable` [constructor](API).
+- The constructor wraps an initialization function that will be executed synchronously every time you call [`subscribe`](API).
+- The initialization function gives you a [`Subscriber`](API) instance to push [notifications](GL) to [consumers](GL).
+- The initialization function is expected to return [teardown](GL) logic, which is either a `() => void`, [`Subscription`](API), or `void` (if there is no teardown to be done).
+- When `complete` or `error` are called on the [`Subscriber`] provided by the initialization function, it will execute the provided [teardown](GL) as soon as possible.
+
+## Overview
+
+Besides consuming observables, another thing you'll likely need to do is create new observables. Creating an observable is relatively straight forward, as it is effectively wrapping a function to be executed later with some guarantees. But more importantly, RxJS provides a host of observable [creation functions](API) that create observables for you in tested ways.
+
+## Creation Functions
+
+A creation function, in RxJS terms, is simply a standalone function provided by RxJS that takes some arguments and returns an [`Observable`](API). There are many examples of these creation functions, but the most common are:
+
+- [`of`](API)
+- [`from`](API)
+- [`concat`](API)
+- [`merge`](API)
+- [`combineLatest`](API)
+- [`forkJoin`](API)
+- [`interval`](API)
+- [`timer`](API)
+- [`defer`](API)
+- [`bindCallback`](API)
+- et al... 
+
+It is highly recommended that you try to use these functions to create your observable. They are well tested, and used in a huge number of projects, and can generally be composed to create most observables you need.
+
+## Observable Construction
```

Comment:
This section gets quite technical rather quickly and this is still pretty early on in the reader's journey.

Consider leaving this introduction paragraph on Observable Construction, so the reader knows there is an alternative. Maybe mention it gets complex. But then move all of the remaining content to a separate section much later in this document and linking to that here.